src/main/java/lotto/domain/Lottos.java

```diff
@@ -0,0 +1,23 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+
+public class Lottos implements Iterable<Lotto>{
```

Comment:
```suggestion
public class Lottos implements Iterable<Lotto> {
```

컨벤션을 위반하셨습니다.
IntelliJ Auto Formatting 기능을 사용하는 습관을 들여보세요.

The auto-formatting shortcut in IntelliJ
* For windows Ctrl + Alt + L.
* For ubuntu Ctrl + Alt + windows + L.
* For Mac Command + Option + L.

Reply:
네! 수정하였습니다.